sandworm/lib/metadata.ml

```diff
@@ -30,13 +48,13 @@ module Bundle = struct
   type t =
     { date : pdate
     ; targets : Target.t list
-    ; has_certificate : (bool[@default false])
-    ; commit : (string option[@default None])
+    ; has_certificate : bool [@default false]
+    ; commit : string [@default "-"]
     }
   [@@deriving yojson]
 
-  let create ~date ~commit ~has_certificate targets =
-    { date; targets; commit; has_certificate }
+  let create ~date ~commit targets =
+    { date; targets; commit; has_certificate = true}
```

Comment:
If you turn the metadata.json into a clean version, I think you can simply remove the `has_certificate` :+1: